app/views/responses/answers/_cascading_editable.html.erb

```diff
@@ -1,12 +1,29 @@
-<%# Show the error messages without keys %>
-<%= content_tag(:div, answer_set.errors.to_hash.values.join(', '), class: 'form-errors') unless answer_set.errors.empty? %>
+<% # Show the error messages without keys %>
+<% if answer_set.invalid? %>
+  <%= content_tag(:div, answer_set.errors.to_hash.values.join(', '), class: "form-errors") %>
+<% end %>
+
+<div class="cascading-selects" id="cascading-selects-<%= context.input_id %>">
+  <% answer_set.children.each_with_index do |answer, index| %>
+    <% level_context = context.add(index) %>
+    <div class="level">
+      <%= hidden_field_tag(level_context.input_name(:id), answer.id) %>
+      <%= hidden_field_tag(level_context.input_name(:type), "Answer") %>
+      <%= hidden_field_tag(level_context.input_name(:questioning_id), answer.questioning_id) %>
+      <%= hidden_field_tag(level_context.input_name(:_relevant), "true") %>
+      <%= hidden_field_tag(level_context.input_name(:_destroy), "false") %>
```

Comment:
Just noticing these should be replaced with a call to `app/views/responses/answers/_hidden_fields.html.erb`. Can you make that happen please? I think it should work out of the box and there should be spec coverage. If you run the response form feature specs and they pass you should be good.